packages/features/liquidity/src/liquidity-leveraged/side-drawers/roll-maturity.tsx

```diff
@@ -49,7 +49,6 @@ export const RollMaturity = () => {
       context={context}
       heading={heading}
       helptext={helptext}
-      enablePrimeBorrow={debt?.tokenType === 'PrimeDebt'}
```

Comment:
I think we should put this `enablePrimeBorrow` prop back everywhere, the logic for when this thing should show up is a bit more specific than what is in `useTransactionApprovals`